### DISCUSSION TITLE
How to load a stylesheet using `with_custom_head` in dioxus 0.6

### DISCUSSION THREAD

**Discussion by photino** [DioxusLabs/dioxus#3344] [Q&A]
In dioxus 0.5, we can use the method [`with_custom_head`](https://docs.rs/dioxus-desktop/latest/dioxus_desktop/struct.Config.html#method.with_custom_head) to load a stylesheet like: 

```html
<link rel="stylesheet" href="public/bulma.min.css" type="text/css">
```
But it does not work in dioxus 0.6. Any ideas on how to load a local stylesheet using `with_custom_head` (not using asset!)?

If we use `asset!`, it seems that it can not run with `cargo run`.

**Comment by tryoxiss** [comment]
Seems like it was replaced by [with_custom_index](https://github.com/DioxusLabs/dioxus/blob/main/examples/custom_html.rs).

**Reply by photino** [reply]
https://github.com/DioxusLabs/dioxus/issues/3349
